packages/docusaurus-theme-classic/src/theme/Tabs/index.tsx

```diff
@@ -109,14 +117,24 @@ function Tabs(props: Props): JSX.Element {
           </li>
         ))}
       </ul>
-      <div className="margin-vert--md">
-        {children.map((tabItem, i) =>
-          cloneElement(tabItem, {
-            key: i,
-            hidden: tabItem.props.value !== selectedValue,
-          }),
-        )}
-      </div>
+
+      {lazy ? (
+        cloneElement(
```

Comment:
isn't the div wrapper missing here?

Reply:
We don't need `div` wrapper if we render only single tab panel. If multiple tab panels are rendered (`!lazy`), then they will be in the `div` wrapper. This is a kind of simplification so as not to create extra `div` elements.

Reply:
Ah I see, didn't notice that you moved the className to the cloned element